index.js

```diff
@@ -17,8 +17,8 @@ console.log();
 
 shell.exec("npm install --save '@gen3/ui-component'");
 shell.exec('npm install');
-shell.cp(`${gen3Dir}/src/App.js`, 'src/App.js');
-shell.cp(`${gen3Dir}/src/App.css`, 'src/App.css');
+shell.exec('rm -rf src');
```

Comment:
Is this standard practice in a node app or will it get flagged for security concerns down the road?

Reply:
Basically this app utilizes `create-react-app` (https://github.com/facebook/create-react-app) and then adds some additional functionality and UI. This line replaces the `src` folder created  by `create-react-app` in the new project with our custom `src` folder. We can talk about alternatives, I wasn't sure what other options there were.